common-arcana.lic

```diff
@@ -109,22 +109,31 @@ module DRCA
     end
   end
 
-  def prepare?(abbrev, mana, symbiosis = false, command = 'prepare', tattoo_tm = false)
+  def prepare?(abbrev, mana, symbiosis = false, command = 'prepare', tattoo_tm = false, runestone_name = nil, runestone_tm = false)
     return false unless abbrev
 
     DRC.bput('prep symb', 'You recall the exact details of the', 'But you\'ve already prepared') if symbiosis
 
-    match = DRC.bput("#{command} #{abbrev} #{mana}", get_data('spells').prep_messages)
+    if runestone_name.nil?
```

Comment:
This is going to break the script right? because `match` isn't defined

Reply:
Ruby keeps the var in scope:

```
if true
	foo = 1
else
	foo = 2
end
 
puts foo
```

> 1

Reply:
Ah ok! It's really confusing nonetheless

Reply:
I'm not a rubyist but my gut tells me its not an intentional use of the language, I'd prefer passing it as an argument then. Not going to raise a fuss about it further though.

Reply:
Yeah, I'm not a fan of this style. I think Rubocop prefers a ternary anyway, something like:

```
match = runestone_name.nil?
              ? DRC.bput(...)
              : DRC.bput(...)
```